avocado/utils/network.py

```diff
@@ -25,31 +25,36 @@
 
 #: Families taken into account in this class
 FAMILIES = (socket.AF_INET, socket.AF_INET6)
+#: Protocols taken into account in this class
+PROTOCOLS = (socket.SOCK_STREAM, socket.SOCK_DGRAM)
```

Comment:
Same comment here about adding an optional parameter for protocols (as for families)